packages/react/test/specs/components/Chat/ChatMessage-test.tsx

```diff
@@ -20,4 +23,14 @@ describe('ChatMessage', () => {
       focusZoneDefinition: (chatMessageBehavior as AccessibilityDefinition).focusZone,
     })
   })
+
```

Comment:
How about another test, that will check that a custom `onMouseEnter` function gets called?

Reply:
It's already tested by `isConformant` test, see `handles events transparently`.